clippy_lints/src/lib.rs

```diff
@@ -503,6 +492,14 @@ pub fn register_plugins(store: &mut rustc_lint::LintStore, sess: &Session, conf:
         "clippy::filter_map",
         "this lint has been replaced by `manual_filter_map`, a more specific lint",
     );
+    store.register_removed(
+        "clippy::pub_enum_variant_names",
+        "use the `avoid_breaking_exported_api` config option to enable the `enum_variant_names` lint for public items",
```

Comment:
A bit more straightforward:

```suggestion
        "set the `avoid_breaking_exported_api` config option to `false` to enable the `enum_variant_names` lint for public items",
```